server/fidelity/content_api_rss.py

```diff
@@ -18,21 +17,13 @@
 WEB_RENDITION = "baseImage"
 CATEGORIES = ("subject_custom",)
 PERMALINK = "Perma_URL"
-BASE_URL = "https://www.fidelityinstitutional.com/"
+BASE_URL = "https://www.fidelityinternational.com/"
 
 
 def get_permalink(item):
-    code = item["_id"][-6:]
-    try:
-        title = item["extra"][PERMALINK] or ""
-        slug = slugify(get_text(title, 'html'))
-    except (KeyError, AttributeError):
-        slug = ""
     return urljoin(
         BASE_URL,
-        "/{lang}/{code}/".format(
-            lang=item.get("language", "en"), code="-".join(filter(bool, [slug, code])),
-        ),
+        f"/editorial/{item['profile']}/{slugify(item.get('headline') or item['name'])}-en5/",
```

Comment:
not sure when they will update production, but atm they are on python 3.5 afaik

Reply:
oh, still 3.5, OK I'll remove the f-string, thanks.